src/re_graph/internals.cljc

```diff
@@ -140,17 +140,20 @@
                                                                "Accept" "application/json"})
                                        (merge {:body (encode payload)
                                                :as :json
+                                               :coerce :always
                                                :async? true
-                                               :throw-exceptions false}))
+                                               :throw-exceptions false
+                                               :throw-entire-message? true}))
                                    (fn [{:keys [status body]}]
                                      (re-frame/dispatch [::http-complete
                                                          instance-name
                                                          query-id
                                                          (if (http/unexceptional-status? status)
                                                            body
                                                            (insert-http-status body status))]))
-                                   (fn [{:keys [status body]}]
-                                     (re-frame/dispatch [::http-complete instance-name query-id (insert-http-status body status)])))]
+                                   (fn [exception]
+                                     (let [{:keys [status body]} (ex-data exception)]
+                                       (re-frame/dispatch [::http-complete instance-name query-id (insert-http-status body status)]))))]
```

Comment:
Although this should never be called, the original function signature was incorrect. The exception is passed to the error callback rather than the raw response.